openfga/v2beta1/openfga.proto

```diff
@@ -0,0 +1,159 @@
+syntax = "proto3";
+
+package openfga.v2beta1;
```

Comment:
do we need to do this now??

Reply:
If we have these changes then we can write the validation for the new model now, as opposed to writing for the old model and then having to update for the new model later.